packages/block-library/src/index.native.js

```diff
@@ -266,7 +266,7 @@ export const registerCoreBlocks = () => {
 		audio,
 		reusableBlock,
 		search,
-		devOnly( embed ),
+		embed,
```

Comment:
**⚠️ I enabled the block just for testing via the installable builds, once we finish the testing we should revert this change.**